frontend/public/components/namespace.jsx

```diff
@@ -521,6 +531,11 @@ class NamespaceBarDropdowns_ extends React.Component {
       items[ALL_NAMESPACES_KEY] = allNamespacesTitle;
     }
 
+    resourceAttributes.namespace = activeNamespace;
+    checkAccess(resourceAttributes).then((res) => {
```

Comment:
This seems like a very odd place to be making this check.
Namespace has no relation to dev-console or rolebindings.